pythonosc/slip_client.py

```diff
@@ -0,0 +1,47 @@
+from typing import Union
+
+from pythonosc.osc_bundle import OscBundle
+from pythonosc.osc_message import OscMessage
+
+import serial
+
+
+class SLIPClient:
+    END = b'\xc0'
+    ESC = b'\xdb'
+    ESC_END = b'\xdc'
+    ESC_ESC = b'\xdd'
+
+    def __init__(self, device, baudrate=9600, **kwargs):
+        self.ser = serial.Serial(device, baudrate=baudrate, **kwargs)
+
+    def send(self, content: Union[OscMessage, OscBundle]) -> None:
+        encoded = self.END
+        encoded += content.dgram.replace(self.ESC, self.ESC + self.ESC_ESC).replace(self.END, self.ESC + self.ESC_END)
+        encoded += self.END
+        self.ser.write(encoded)
+
+    def receive(self):
```

Comment:
can you please add some docs on what to expect as return type please (OscMessage)